Jenkinsfile

```diff
@@ -48,6 +49,79 @@ pipeline {
 
     stage('Tests') {
       parallel {
+        stage('Prepare Windows 2016 with containers') {
+          agent { label 'executor-windows-2016-containers' }
+          stages {
+            stage('Configure Windows Node'){
+              steps {
+                powershell '.\\expose-daemon.ps1'
+                script {
+                  env.WINDOWS_IP = powershell(returnStdout: true, script:  '(curl http://169.254.169.254/latest/meta-data/local-ipv4).Content').trim()
```

Comment:
is there any risk for having this IP hardcoded here?

Reply:
@izgeri I think it's fine since that IP is publicly known as the place to get metadata on and from any given EC2 instance https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html